public/sass/components/_query_part.scss

```diff
@@ -1,10 +1,6 @@
 .query-part {
   background-color: $tight-form-func-bg;
 
-  &.unknown-function {
```

Comment:
Is it possible this class might still be used by other query editors / plugins? 🤔

Reply:
I don't think so. I added it recently for a new features specifically for Graphite: https://github.com/grafana/grafana/pull/32635